src/bases/bdm3dspace.jl

```diff
@@ -3,6 +3,7 @@ mutable struct BDM3DBasis{T,M,P} <: Space{T}
     fns::Vector{Vector{Shape{T}}}
     pos::Vector{P}
 end
+redefine_geometrie(basis::BDM3DBasis{T,M,P},geo::F) where {T,M,P,F}= BDM3DBasis{T,F,P}(geo,basis.fns,basis.pos)
```

Comment:
Are you using this function?

Reply:
Yes, to add the trace mesh to already defined bases, because the infinitisimal translation of the test function is done in the assembly